assets/js/app.js

```diff
@@ -120,27 +119,16 @@ function initializeAddWidget(grid) {
 }
 
 function initializeChangeHandler(grid) {
-    grid.on("change", function (event, widgets) {
-
-        if (event.target.classList.contains("lle-dashboard-input-title")) {
-            // it's the title edition, ignore it
-            return;
-        }
-
-        // Create EventHandler for widget update
-        for (let widget of widgets) {
-
-            let url = Routing.generate("update_widget", {
-                id: widget.id,
-                x: widget.x,
-                y: widget.y,
-                width: widget.w,
-                height: widget.h,
-            });
+    window.addEventListener('resize', function () {
+        location.reload();
```

Comment:
you reload the page every time the user resizes his browser ? that's a no for me